src/lib/sort/sort.md

```diff
@@ -0,0 +1,19 @@
+To add sorting behavior and styling to a set of table headers, add the `<md-sort-header>` component
```

Comment:
The overview should be structured like
1. Single sentence that summarizes the components, e.g.: "The `mdSort` and `md-sort-header` are used, respectively, to add sorting state and display to tabular data."
2. overview example
3. Basic use
4. Additional sections on configuration, alternate behaviors, etc.
5. Accessibility notes for this component